### DISCUSSION TITLE
Can the search bar be docked at the bottom of the window?

### DISCUSSION THREAD

**Discussion by wojdyr** [zed-industries/zed#19623] [Feature Requests]
I'm new to Zed and, coming from Vim, I'm used to looking down after pressing `/`.
Is it possible to configure the position of the search bar?

**Comment by machin3io** [comment]
Bumping this. Looking up feels awkwardly hard 😅 Dimming the background would be nice too, easymotion style.

**Comment by wojdyr** [comment]
I guess I should have make it a feature request, not Q&A.
Added a feature request now: https://github.com/zed-industries/zed/issues/20159
